deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -212,6 +212,10 @@ def _execute_compaction(
         logger.info("No input deltas found to compact.")
         return None, None, None
 
+    if not uniform_deltas:
+        logger.info("No uniform deltas found to compact.")
+        return None, None, None
+
```

Comment:
This is a valid scenario, we have to reference all files and write back the compacted shard with updated stream position. That would correctly update the completeness. if we return like this, the completeness of the table is never updated.